src/screens/Assets/WalletView.js

```diff
@@ -297,10 +302,10 @@ class WalletView extends React.Component<Props, State> {
           {activeTab === COLLECTIBLES && (
             <CollectiblesList
               collectibles={this.getFilteredCollectibles()}
-              searchQuery={query}
-              navigation={navigation}
-              activeAccountAddress={activeAccountAddress}
-            />)}
+              onCollectiblePress={this.handleCollectiblePress}
+              isSearching={!!query}
```

Comment:
Now it isn't changing much, but I guess, technically it should be:
```suggestion
              isSearching={query.length >= MIN_QUERY_LENGTH}
```
(as only then collectibles are being filtered)